### PR TITLE
Style guide | Add new colors #2767

### PR DESCRIPTION
**Pull request by Sjones352**
Connects #2767 

screen shot 2017-08-14 at 12 39 14 pm

### CODE REVIEW

File: client/app/containers/StyleGuide/StyleGuideColors.jsx [hunk 1]
```diff
@@ -156,6 +166,19 @@ export default class StyleGuideColors extends React.Component {
       )}
       </div>
 
+      <h3>Reader Categories</h3>
+
+      <div className="sg-colors-swatches">
+      {this.Reader.map((name, i) =>
+        <div className="sg-colors-swatch" key={name + i}>
```
Comment (NickHeiner) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: Using the index in the key is a bad idea. It means that if we re-order the list, we'll have to re-render. Please just use `name`.

File: client/app/containers/StyleGuide/StyleGuideColors.jsx [hunk 2]
```diff
@@ -24,8 +24,11 @@ export default class StyleGuideColors extends React.Component {
     Dispatch: '#844e9f',
     'eFolder Express': '#F0835e',
     Feedback: '#73e5d4',
-    Certification: '#459fd7',
-    Reader: '#417505'
+    Reader: '#417505',
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: I just realized that when the first Certification hex code was deleted, the deleted color was for the _logo_ and not the Reader tag. Please add the old hex code back

File: client/app/containers/StyleGuide/StyleGuideColors.jsx [hunk 3]
```diff
@@ -24,8 +24,11 @@ export default class StyleGuideColors extends React.Component {
...
+    Hearing: '#fdc231',
+    Procedural: '#5a94ec',
+    Medical: '#ff6868',
+    Certification: '#3ad2cf'
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: Rename this key as `CertificationTag` (to differentiate that it is a reader tag color)

File: client/app/containers/StyleGuide/StyleGuideColors.jsx (hunk 1 above)
Comment (Sjones352) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: Fixed

File: client/app/containers/StyleGuide/StyleGuideColors.jsx [hunk 4]
```diff
@@ -147,15 +157,28 @@ export default class StyleGuideColors extends React.Component {
...
+      )}
+      </div>
+
+      <h3>Reader Categories</h3>
+
+      <div className="sg-colors-swatches">
+      {this.Reader.map((name) =>
+        <div className="sg-colors-swatch" key={name}>
           <div style={{ background: this.Colors[name] }}></div>
           <b>{this.Colors[name]}</b>
```
Comment (mmlumba) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: Add a conditional that changes this value to be `Certification` if the key chosen is `CertificationTag`

Comment (Sjones352) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: Fixed

File: client/app/containers/StyleGuide/StyleGuideColors.jsx (hunk 3 above)
Comment (Sjones352) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: Fixed

File: client/app/containers/StyleGuide/StyleGuideColors.jsx (hunk 2 above)
Comment (Sjones352) [inline client/app/containers/StyleGuide/StyleGuideColors.jsx]: Fixed

**Pull request merged**